Applications/Games/Overwatch/Online/script.js

```diff
@@ -0,0 +1,24 @@
+include(["Engines", "Wine", "QuickScript", "OnlineInstallerScript"]);
+include(["Engines", "Wine", "Verbs", "vcrun2015"]);
+include(["Engines", "Wine", "Verbs", "corefonts"]);
+
+new OnlineInstallerScript()
+    .name("Overwatch")                      
+    .editor("Blizzard")                              
+    .applicationHomepage("http://www.playoverwatch.com/")  
+    .author("ImperatorS79")                     
+    .url("https://eu.battle.net/download/getInstaller?os=win&installer=Overwatch-Setup.exe")                      
+    //The checksum is different each time you download
+    .wineVersion("LATEST_STAGING_VERSION")
```

Comment:
You need to remove the `"` here. `LATEST_STAGING_VERSION` is a constant variable

Reply:
Done